### PR TITLE
Fix score drop issue

### PR DESCRIPTION
**Pull request by JihwanEom**
- Updated test pipeline & config following OTE convention.

### CODE REVIEW

File: recipes/stages/_base_/models/segmentors/seg_class_incr.py [hunk 1]
```diff
@@ -67,5 +67,9 @@
                       sampler=dict(type='MaxPoolingPixelSampler', ratio=0.25, p=1.7),
                       loss_weight=1.0)
              ])
-    ]
+    ],
+    test_cfg=dict(
+        mode='whole',
+        output_scale=10.0,
+    ),
```
Comment (goodsong81) [inline recipes/stages/_base_/models/segmentors/seg_class_incr.py:74]: Could you explain about this setting?

**Review by harimkang — APPROVED** [review]
Where did you refer to the output_scale value? And how much difference does it make?

**Comment by JihwanEom** [comment]
Hi, I refered OTEsegmentation lite-hrnet template. https://github.com/openvinotoolkit/training_extensions/blob/class-incr-learning-validation/external/mmsegmentation/configs/custom-sematic-segmentation/ocr-lite-hrnet-18/model.py#L131

I could see significant difference when I adopt output_scale, **0.0779 -> 0.3806**, exactly same value with OTEseg evaluation.

**Review by chuneuny-emily — APPROVED** [review]
It looks good.

**Review by goodsong81 — APPROVED** [review]

File: recipes/stages/_base_/models/segmentors/seg_class_incr.py (hunk 1 above)
Comment (JihwanEom) [reply recipes/stages/_base_/models/segmentors/seg_class_incr.py:74]: In lite-hrnet OCR head architecture, FCN first output logits and OCR get the logits as its input. (FCN -> OCR). 
At this time, if there exists output_scale config, "FCN logit * output_scale" becomes OCR head input. For more detailed information, please refer here : https://github.com/openvinotoolkit/mmsegmentation/blob/571a0d38069588787916c682d94888614352972b/mmseg/models/segmentors/cascade_encoder_decoder.py#L99
